### DISCUSSION TITLE
Help with grepping over notify notifications.

### DISCUSSION THREAD

**Discussion by hfytr** [ibhagwan/fzf-lua#1293] [General]
So i use nvim-notify: https://github.com/rcarriga/nvim-notify
And I have this mapping```

map("n", "<leader>sn", function()
    local history = require("notify").history()
    require("fzf-lua").fzf_exec(history)
end, opts)
```
but no dice. whenever i try, i get the error: ```

  E5108: Error executing lua: ...bwdw/.local/share/nvim/lazy/fzf-lua/lua/fzf-lua/c 
 core.lua:83: assertion failed! 
 stack traceback: 
 [C]: in function 'assert' 
 ...bwdw/.local/share/nvim/lazy/fzf-lua/lua/fzf-lua/core.lua:83: in function 'co 
 ontents_from_arr' 
 ...bwdw/.local/share/nvim/lazy/fzf-lua/lua/fzf-lua/core.lua:137: in function 'f 
 fzf_exec' 
 /home/fbwdw/.config/nvim/lua/configs/fzf-lua.lua:14: in function </home/fbwdw/. 
 .config/nvim/lua/configs/fzf-lua.lua:12> 
```

**Comment by ibhagwan** [comment]
I don’t know what is the contents of `require("notify").history()`, fzf_exec can work with different types of contents but in your case you’d need to convert the result set from notify to a table of strings, use `vim.print` to explore the contents of the notify history and make the appropriate conversions.

**Comment by peterbenjamin-kh** [comment]
> And I have this mapping
> ```lua
> map("n", "sn", function()
>   local history = require("notify").history()
>   require("fzf-lua").fzf_exec(history)
> end, opts)
> ```

nvim-notify provides a direct way of doing this:

```lua
require("notify.integrations").pick()
```
